backend/database/migrations/1669168791-create-profile.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('profile', {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true

```

Comment:
allowNull: false.

Reply:
Primary key is never null so it's implicitly done.